dotnet/src/Microsoft.AutoGen/RuntimeGateway.Grpc/Abstractions/AgentTypes.cs

```diff
@@ -0,0 +1,18 @@
+// Copyright (c) Microsoft Corporation. All rights reserved.
+// AgentTypes.cs
+using Microsoft.AutoGen.Core;
+namespace Microsoft.AutoGen.RuntimeGateway.Grpc.Abstractions;
+public sealed class AgentTypes(Dictionary<string, Type> types)
```

Comment:
I thought you mentioned this was being moved to tests?

Reply:
my last commit hadn't sync'd to GH yet before I closed my laptop evidently